tests/wpt/metadata/XMLHttpRequest/responsexml-document-properties.htm.ini

```diff
@@ -3,12 +3,6 @@
   [domain]
     expected: FAIL
 
-  [URL]
```

Comment:
Huh, I'm confused why this test result changed. The test doesn't use responseURL at all.